org.sofproject.gst.topo/build.properties

```diff
@@ -2,4 +2,7 @@ source.. = src/
 output.. = bin/
 bin.includes = META-INF/,\
                .,\
-               plugin.xml
+               plugin.xml,\
+               lib/jackson-annotations-2.10.1.jar,\
```

Comment:
Do not import external binaries to the plug-in repository. Add a dependency to an existing Eclipse plug-in(s) to the manifest (and adjust the documentation if any of them is not a part of core bundle,  see how the dependency on GEF is handled).